### PR TITLE
adcm_is_upgradable fixture added

### PR DESCRIPTION
**Pull request by Sealwing**

### CODE REVIEW

**Review by skhomuti — APPROVED** [review]

File: src/adcm_pytest_plugin/fixtures.py [hunk 1]
```diff
@@ -294,37 +297,64 @@ def _remove_hosts(adcm_cli: ADCMClient):
...
+def adcm_is_upgradable_ms(request) -> bool:
+    """Set flag that controls either ADCM will be upgradable or not"""
+    return _adcm_is_upgradable(request)
+
+
+@allure.title("[FS] ADCM upgradable flag")
+@pytest.fixture()
+def adcm_is_upgradable_fs(request) -> bool:
+    """Set flag that controls either ADCM will be upgradable or not"""
+    return _adcm_is_upgradable(request)
```
Comment (dgusakov) [inline src/adcm_pytest_plugin/fixtures.py:357]: Why not use a single session-scoped fixture?

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: src/adcm_pytest_plugin/fixtures.py (hunk 1 above)
Comment (Sealwing) [reply src/adcm_pytest_plugin/fixtures.py:357]: In that case we'll alter all levels of adcm fixture, so each adcm_fs will be upgradable

Comment (dgusakov) [reply src/adcm_pytest_plugin/fixtures.py:357]: Nope.
Only adcm_xx fixtures of the given module or package.

Comment (Sealwing) [reply src/adcm_pytest_plugin/fixtures.py]: Got it, fixed

**Review by dgusakov — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**